### PR TITLE
Adds new tips

### PR DESCRIPTION
**Pull request by Wer6**
## About The Pull Request
Adds tips

## Why It's Good For The Game
Warns marines and xenos about the dangers they might face.

## Changelog
:cl:
add: Added new tips
/:cl:

### CODE REVIEW

File: strings/tips/xeno.txt [hunk 1]
```diff
@@ -32,8 +32,8 @@ As a xeno, remember that neurotox injected by defilers, sentinels, panthers, and
 As a xeno, you can use the crawl through vent verb found in the xeno tab to quickly find and use vents in the floor without having to alt-click them. 
 As a xeno, you can crawl through holes in walls made by acid by SpecialClicking (Default: CTRL) on them or click-dragging yourself on them. If you're too big to pass through, you'll damage the wall instead.
 As a xeno, you can climb over window frames by SpecialClicking (Default: CTRL) on them or click-dragging yourself on them.
-As a xeno, do not camp the landing zones prior to the marines building their defenses.
+As a xeno, do camp the landing zones prior to the marines building their defenses.
```
Comment (Hughgent) [inline strings/tips/xeno.txt:35]: I'm iffy on this one

**Review by Hughgent — APPROVED** [review]
Otherwise ok

File: strings/tips/xeno.txt (hunk 1 above)
Comment (Wer6) [reply strings/tips/xeno.txt:35]: i can remove it easy. that line is sorta misleading. maybe just the removal of that line?

**Comment by spookydonut** [comment]
@Wer6 Please use sensible PR names and changelog entries.

File: strings/tips/xeno.txt (hunk 1 above)
Comment (TiviPlus) [reply strings/tips/xeno.txt:35]: What
Do not camp wtf fobcamp bad

**Review by Hughgent — APPROVED** [review]

File: strings/tips/marine.txt [hunk 2]
```diff
@@ -30,6 +30,17 @@ To throw a grenade, Activate (Default: Z) the grenade while in the active hand.
 Quickly store items to the container (bags, satchels, belts, pouches) by pressing the Quick Equip (Default: E) key while holding an item and you are looking at a container.
 Press Quick Equip (Default: E) while you are not holding an item to the active hand to draw the weapon from whatever preferred slot (Default: Suit Storage, can be customized at the Preferences tab) you selected, otherwise you will pull out any item from a container.
 When captured by the xenomorphs, infected and you think you are not going to make it out alive, use a pocket pistol on yourself to prevent the chestburster from popping out. A dead marine is better than a popped marine.
+Getting hit by a xeno's claws can cause injury, or even death.
+Getting hit by bullets can cause injury, or even death.
+Getting hit by explosions can cause injury, or even death.
+Getting set on fire can cause injury, or even death.
+Stepping on acid can cause injury, or even death.
+Getting infected by a facehugger can cause injury, or even death.
```
Comment (XSlayer300) [inline strings/tips/marine.txt:38]: Not gonna lie, these tips are too obvious for me to be called them 'tips'.

**Pull request closed without merging**